src/mozillavpn.cpp

```diff
@@ -102,7 +102,9 @@ MozillaVPN::MozillaVPN() : m_private(new Private()) {
       break;
     }
   }
-  AdjustHandler::initialize(adjustProxy->serverPort());
+  connect(this, &MozillaVPN::stateChanged, [adjustProxy]() {
```

Comment:
OK! 1 extra step, can we move the port creation inside the AdjustHandler::initialize?